OpenRA.Mods.Common/Scripting/Properties/ProductionProperties.cs

```diff
@@ -86,7 +86,7 @@ public CPos RallyPoint
 				if (rp.Path.Count > 0)
 					return rp.Path.Last();
 
-				var exit = Self.FirstExitOrDefault();
+				var exit = Self.NearestExitOrDefault();
```

Comment:
Maybe `RandomExitOrDefault`, then `NearestExitOrDefault` could make `nearestTo` a required parameter?

Reply:
Or, since randomization isn't wanted here, we could call `NearestExitOrDefault` with `Self.CenterPosition`?

Reply:
The latter is probably the better idea.